app/presenters/explorer_presenter.rb

```diff
@@ -163,7 +163,7 @@ def rx(data)
   end
 
   def self.open_window(url)
-    new(:mode => 'window', :open_url => url)
+    new({:mode => 'window', :open_url => url})
```

Comment:
Curious why was this necessary?

Reply:
Yes, kind of strange as I was having problem when rail tried to do the implicit conversion to dictionary. It only started working when I made it explicit like this or passed a dictionary variable 
my_options = {:mode => 'window', :open_url => url}.
The init function could then get the both the mode and the open_url.

Reply:
None of the rest of these initializations does this so I'm thinking that wasn't actually the issue: https://github.com/ManageIQ/manageiq-ui-classic/pull/8149/files#diff-c21bb487c2cf9ad93d9c7b6c7667d83565b5ae29d8086f67df6cf19b1330b2edL48-L66

In ruby you don't have to put `{}` around a hash argument to a method unless it is mixed with positional arguments, which this is not (https://github.com/ManageIQ/manageiq-ui-classic/pull/8149/files#diff-c21bb487c2cf9ad93d9c7b6c7667d83565b5ae29d8086f67df6cf19b1330b2edR68)